contrib/google.golang.org/api/internal/gen_endpoints/main.go

```diff
@@ -175,96 +197,80 @@ func handleMethod(def *APIDefinition, method *APIMethod) ([]Endpoint, error) {
 	if !strings.HasPrefix(path, "/") {
 		path = "/" + path
 	}
-
 	uritpl, err := uritemplate.New(path)
 	if err != nil {
 		return nil, err
 	}
-	return []Endpoint{{
+	return &Endpoint{
 		Hostname:     hostname,
 		HTTPMethod:   method.HTTPMethod,
 		PathTemplate: path,
 		PathMatcher:  uritpl.Regexp(),
 		ServiceName:  "google." + def.Name,
 		ResourceName: method.ID,
-	}}, nil
+	}, nil
 }
 
-func downloadGoogleApiSrc() string {
-	zipUrl := downloadUrl()
+func downloadGoogleAPISrc() (string, error) {
+	zipUrl := googleAPIClientURL()
```

Comment:
ooops I missed this one before
```suggestion
	zipURL := googleAPIClientURL()
```